File: app/src/main/java/com/test/test3app/activity/ZRouter.java
package com.test.test3app.activity;

import android.app.Activity;

/**
 * created by zhaoyuntao
 * on 27/03/2023
 */
public class ZRouter {
    private String name;
    private Class<? extends Activity> clazz;

    public ZRouter(String name, Class<? extends Activity> clazz) {
        this.name = name;
        this.clazz = clazz;
    }

    public String getName() {
        return name;
    }

    public Class<?> getActivity() {
        return clazz;
    }
}

File: app/src/main/java/com/test/test3app/appbase/ZRouter.kt
package com.test.test3app.appbase

import android.app.Activity
import android.view.View
import com.test.test3app.activity.MainActivity1_unknown

/**
 * created by zhaoyuntao
 * on 05/04/2022
 * description:
 */
public data class ZRouter(var name:String, var activity: Class<out Activity>) {
}